src/psmove_orientation.c

```diff
@@ -27,7 +27,9 @@
  * POSSIBILITY OF SUCH DAMAGE.
  **/
 
-
+#ifdef _MSC_VER
+#define __func__ __FUNCTION__
```

Comment:
`__func__` is not the same as `__FUNCTION__`, but if MSVC doesn't have it, maybe change it all occurrences of `__func__` to `__FUNCTION__` and not have per-compiler handling of that, and this `#ifdef` in all files?